### PR TITLE
Features/deploy rebrand

### PR DESCRIPTION
**Pull request by baktun14**
- Refactored the nextjs pages into app router
- Refactored material ui components into tailwind + shadcn
- Pages that are mostly refactors
  - Home, Deployments, Providers
- I added that MUI also work with Tailwind, so we can gradually refactor the MUI components or keep using them.
---
### **Important**
This PR is simply to merge a big chunk back into the collab branch and to be able to split the remaining work between potentially more devs. So not everything is done and a lot of stuff is still broken.

### CODE REVIEW

File: api/src/services/external/apiNodeService.ts [hunk 1]
```diff
@@ -478,7 +488,22 @@ export async function getAddressDeployments(owner: string, skip: number, limit:
...
+          const provider = providers.find((p) => p.owner === lease.lease.lease_id.provider);
+          return {
+            id: lease.lease.lease_id.dseq + lease.lease.lease_id.gseq + lease.lease.lease_id.oseq,
+            owner: lease.lease.lease_id.owner,
+            provider: provider,
+            dseq: lease.lease.lease_id.dseq,
+            gseq: lease.lease.lease_id.gseq,
+            oseq: lease.lease.lease_id.oseq,
+            state: lease.lease.state,
+            price: lease.lease.price
```
Comment (Redm4x) [inline api/src/services/external/apiNodeService.ts:504]: Will need to add to swagger doc, I can do it in another branch.

File: deploy-web/src/app/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      path: "./fonts/Satoshi-Variable.ttf",
+      weight: "400",
+      style: "normal"
+    }
+  ]
+  // variable: "--font-sans"
+});
+
+export const metadata: Metadata = {
+  title: "Akash Network Stats",
```
Comment (Redm4x) [inline deploy-web/src/app/layout.tsx]: Will need to put correct metadata

**Review by Redm4x — APPROVED** [review]
I went through really quickly just to get an idea of the changes. Didn't really put comments since it's a WIP and there is still lots of TODOs and things to cleanup.

File: deploy-web/src/app/layout.tsx (hunk 2 above)
Comment (baktun14) [reply deploy-web/src/app/layout.tsx]: Yea I'll change it

**Comment by baktun14** [comment]
> I went through really quickly just to get an idea of the changes. Didn't really put comments since it's a WIP and there is still lots of TODOs and things to cleanup.

Sounds good, yea a lot of stuff is unfinished and needs cleanup

**Pull request merged**